docs/http-api.md

```diff
@@ -2,43 +2,42 @@
 title: HTTP API
 ---
 
-Before you can start calling the server APIs you need to take care of the following steps first:
+Before you can start calling the server APIs, you need to take care of the following steps:
 
 1. [Create a free account on statsig.com](#step1)
-2. Get an API key from the admin console
-3. Issue API requests
+2. [Get an API key from the admin console](#step2)
+3. [Issue API requests](#step3)
 
-<a name="step1"></a>
+---
+
+### Step 1 - Create a free account on the [Statsig sign-up page](https://console.statsig.com/sign_up)
```

Comment:
we should add a callout that you should avoid the httpapi and use a relevant SDK where possible for a much more performant integration